plrust-trusted-pgx/Cargo.toml

```diff
@@ -1,10 +1,9 @@
 [package]
-name = "trusted-pgx"
-# version set to match that of the pgx version it sits on top of, rather than matching plrust's version
-version = "0.7.2"
+name = "plrust-trusted-pgx"
+version = "0.0.0-do-not-use"
 authors = ["TCDI <opensource@tcdi.com>"]
 edition = "2021"
-license = "PostgreSQL Open Source License"
+license = "MIT"
```

Comment:
I had to do this to appease publishing to creates.io.

Reply:
you could also do `license-file = "LICENSE.md"` I think? (Possibly might need to copy or symlink the license file into the same directory)